python/lsst/pipe/analysis/utils.py

```diff
@@ -222,6 +223,30 @@ def __call__(self, catalog):
         cosDec2 = np.cos(catalog[self.declination2]) if self.declination2 is not None else 1.0
         return (first*cosDec1 - second*cosDec2)*(1.0*afwGeom.radians).asArcseconds()*self.unitScale
 
+class AngularDistance(object):
+    """Functor to calculate the Haversine angular distance between two points
```

Comment:
This docstring confused me.  This really computes the distance between two columns right?

Reply:
Well, from Wikipedia:  "The haversine formula determines the great-circle distance between two points on a sphere given their longitudes and latitudes."  I do initialize the class using 4 columns (with names `raStr1`, `decStr1`, `raStr2`, `decStr2`) that represent the two points between which we want to compute this distance.  I've updated the docstring to make this more clear.